requirements.txt

```diff
@@ -37,4 +37,5 @@ service_identity
 setuptools==65.5.1
 werkzeug==2.2.3
 wheel
-zope.interface==5.4.0
\ No newline at end of file
+zope.interface==5.4.0
+pandas<2.0.0
```

Comment:
Maybe we should keep the dependencies separate in the utils to be able to build a lean container image in Krkn-hub for running the chaos-recommender? Thoughts?